### PR TITLE
add JSONLD schema DSS and Schema DSS_model

### PR DESCRIPTION
**Pull request by mlabadie**

### CODE REVIEW

**Comment by pradal** [comment]
This PR is superseed by #3 
@treinar : can you close this PR without merging it?

File: DSS_metadata/DSS_model.jsonld [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "@context":{
+        "schema": "http://schema.org/",
+        "ipm": "https://raw.githubusercontent.com/H2020-IPM-openalea/formats/master/schema_Terms_from_VIPS.md/",
```
Comment (pradal) [inline DSS_metadata/DSS_model.jsonld]: Use H2020-IPM-Decisions instead

File: schema_Terms_from_VIPS.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Json-LD Documentation
+
+## Sources documentations
+
+Json-ld: <https://json-ld.org/>
```
Comment (pradal) [inline schema_Terms_from_VIPS.md]: Json-ld: <https://json-ld.org>

File: schema_Terms_from_VIPS.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Json-ld documentation: <https://json-ld.org/learn.html>
+schema: <http://www.schema.org>
+
+## JSON-LD Context creation
+
+1. DSS_schema context (VIPS): DSS_schema.jsonld
```
Comment (pradal) [inline schema_Terms_from_VIPS.md]: VIPS need to be an example not in the definition of the schema

**Review by pradal — CHANGES_REQUESTED** [review]

**Comment by treinar** [comment]
As requested by @pradal, this pull request is now closed.

**Pull request closed without merging**